test/e2e/tests/signature-request.spec.js

```diff
@@ -8,17 +8,45 @@ const {
 } = require('../helpers');
 const FixtureBuilder = require('../fixture-builder');
 
+const ganacheOptions = {
+  accounts: [
+    {
+      secretKey:
+        '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+      balance: convertToHexValue(25000000000000000000),
+    },
+  ],
+};
+
+async function verifyAndAssertSignTypedDataV4(driver) {
```

Comment:
Why is it only the typed tests have these helpers to verify and approve but not `ethSign` or `personal`? Are they applicable to both? Could they go in a common location to be used across all three?

Reply:
I've improved the signature request to use two templates one for the normal signature request and the other to support multiple operations. I also added some helpers in `ethSign` and `personal` but when I tried to expand the template that I've created to signature request it got really confusing and fragile.